addons/l10n_at/__manifest__.py

```diff
@@ -3,7 +3,7 @@
     'name': 'Austria - Accounting',
     'version': '3.0',
     'author': 'WT-IO-IT GmbH, Wolfgang Taferner',
```

Comment:
```suggestion
    'author': 'WT-IO-IT GmbH, Wolfgang Taferner (https://www.wt-io-it.at)',
```
Please keep the link to the author's website. I feel like it's a nice way to thank them for their contributions to let them keep that visibility

Same applies for every localization we had a  link to the author's website